RELEASE-NOTES.md

```diff
@@ -0,0 +1,45 @@
+# Preparing a Renku release
+
+These are the steps that need to be followed to make a new release of Renku.
+
+1. Make a list of new component versions to include in the release including *renku-python*.
+
+2. If a new *renku-python* version is needed:
+
+   a. Check if images in  are built using the desired version of *renku-python*. If not, modify `.travis.yml` file to include the desired version. Keep one line per minor version with the latest patch.
```

Comment:
mention which repo you are talking about

Reply:
cut-paste error, thanks for noticing!